packages/xrpl/src/Wallet/index.ts

```diff
@@ -318,6 +321,7 @@ class Wallet {
     this: Wallet,
     transaction: Transaction,
     multisign?: boolean | string,
+    customDefinitions: DefinitionContents = DEFINITIONS,
```

Comment:
```suggestion
    customDefinitions: DefinitionContents = undefined,
```

Reply:
Kept the default values at the low level (`binary.ts` and `binary-parser.ts` primarily) and then removed the need for it in the higher level functions (like `encode` and `decode`).